### PR TITLE
F009 main content container

### PR DESCRIPTION
**Pull request by grzegorzbors**

### CODE REVIEW

**Review by rwedzony — APPROVED** [review]

File: src/layouts/Layout.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import Header from "./Header";
-import Content from "./Content";
-import Footer from "./Footer";
+import Header from "../components/Header";
+import Content from "../components/Content";
+import Footer from "../components/Footer";
+
 import { BrowserRouter as Router } from "react-router-dom";
 
 const Layout = () => {
```
Comment (mwojtow) [inline src/layouts/Layout.js:7]: You only need one <Router> on app core for basic routing, if it's in App you don't need it here

File: src/components/Content.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Home, About, Services, Team, Appointment, FAQ, Contact, PersDataProt, PrivacyPolicy } from "../pages/index";
+
+import { Switch, Route } from "react-router-dom";
+import { routes } from "../routing/routes";
+
+const Content = () => {
+  return (
+    <>
```
Comment (mwojtow) [inline src/components/Content.js:8]: Fragment is not needed if you return one node, like <Switch> here.  Please check other places and remove if not needed.

File: src/pages/PersDataProt.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const PersDataProt = () => {
```
Comment (mwojtow) [inline src/pages/PersDataProt.js:1]: don't be afraid and use full name - now you know what it is, but in few weeks you will be like, wtf is that Prot in name ;) it's also good for other developers to read and understand code

File: src/components/Content.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Home, About, Services, Team, Appointment, FAQ, Contact, PersDataProt, PrivacyPolicy } from "../pages/index";
+
+import { Switch, Route } from "react-router-dom";
+import { routes } from "../routing/routes";
+
+const Content = () => {
```
Comment (mwojtow) [inline src/components/Content.js:6]: Name Content is confusing for me, I would expect to have here some JSX which is rendered as main part of page/app. Maybe better would be AppRouting/AppRouter ? Name that clearly say that this file contains routing.

**Review by mwojtow — CHANGES_REQUESTED** [review]

File: src/components/Content.js (hunk 2 above)
Comment (grzegorzbors) [reply src/components/Content.js:8]: Removed fragments.

Comment (grzegorzbors) [reply src/components/Content.js:8]: Removed fragments.

File: src/pages/PersDataProt.js (hunk 3 above)
Comment (grzegorzbors) [reply src/pages/PersDataProt.js:1]: Name changed.

File: src/layouts/Layout.js (hunk 1 above)
Comment (grzegorzbors) [reply src/layouts/Layout.js:7]: Kept routing in App only.

File: src/components/Content.js (hunk 4 above)
Comment (grzegorzbors) [reply src/components/Content.js:16]: Changed name to AppRouting.

File: src/components/AppRouting.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  Home,
+  About,
+  Services,
+  Team,
+  Appointment,
+  FAQ,
+  Contact,
+  PersonalDataProtection,
+  PrivacyPolicy,
+} from "../pages/index";
```
Comment (mwojtow) [inline src/components/AppRouting.js:11]: when importing from "./xxxx/index" you don't need "index", it's default so "./xxx" is enough :)

**Review by mwojtow — APPROVED** [review]

File: src/components/AppRouting.js (hunk 5 above)
Comment (grzegorzbors) [reply src/components/AppRouting.js:11]: Changed import.

**Pull request merged**